quantum_systems/time_evolution_operators/operator.py

```diff
@@ -73,7 +81,7 @@ def u_t(self, current_time):
             time-point.
         """
 
-        return self._system.u
+        return 0
```

Comment:
Why is this returning 0 instead of self._system.u?

Reply:
See https://github.com/Schoyen/quantum-systems/pull/57#discussion_r617476181.